### DISCUSSION TITLE
How do I run non-base Flux models?

### DISCUSSION THREAD

**Discussion by gruevy** [lllyasviel/stable-diffusion-webui-forge#2188] [General]
I can get the base flux models to run just fine. Very simple, they work out of the box. For non-base Flux models I've been told I need to manually select the following in the VAE box:

ae.safetensors (which is just a renamed ae.sft, i guess?)
clip_l.safetensors
t5xxl_fp16.safetensors

The UI keeps crashing on me with no errors. I have no idea what I'm doing wrong. I'm trying to use the latest version of this: https://civitai.com/models/141592?modelVersionId=992642

Here's my settings:  
![image](https://github.com/user-attachments/assets/37593e79-320f-4725-bb9f-a85ff1aa6d83)

Any tips? This happened last time I tried a custom flux model, too.

**Comment by ErmMonzon** [comment]
Maybe change your swap location to shared and lower your GPU weight (try 17-18000). Should be fine as I can run the same model with just 8GB of VRAM.

**Reply by gruevy** [reply]
Tried that, still not working. Glad to know it's possible, though. If it's nothing obvious, I might just have to reinstall

**Reply by ErmMonzon** [reply]
Do you not see any error messages in the command window? I'm literally running it right now, only difference is I'm using FP8 version of the model as the T5xxl-fp8 text encoder

![image](https://github.com/user-attachments/assets/d9818e60-e992-49ab-9522-75f769b789bf)

**Reply by gruevy** [reply]
Well I got it working now. I used the fp8 encoder like you did, and that didn't work. Played around and discovered that it works if I turn my GPU weight down to 8gb. It's slow, and it feels dirty having a 4090 and only using 8gb of ram, but it works. Not sure why. But thanks for the solution!
